pulsar-client-tools/src/main/java/org/apache/pulsar/client/cli/CmdProduce.java

```diff
@@ -114,6 +120,15 @@
     @Parameter(names = { "-k", "--key"}, description = "message key to add ")
     private String key;
 
+    @Parameter(names = { "-vs", "--value-schema"}, description = "Schema type (can be bytes,avro,json,string...)")
+    private String valueSchema = "bytes";
+
+    @Parameter(names = { "-ks", "--key-schema"}, description = "Schema type (can be bytes,avro,json,string...)")
+    private String keySchema = "string";
+
+    @Parameter(names = { "-kvet", "--key-value-encoding-type"}, description = "Key Value Encoding Type (can be none,separated,inline)")
+    private String keyValueEncodingType = "none";
```

Comment:
I don't think we should introduce `none` here. The default value should be set to be `null`. If `keyValueEncodingType` is not specified, you can still use the keySchema to serialize the key to `bytes` and use `keyBytes` to set the key.

Reply:
makes sense to me.
I have removed that value.
now if you do not pass explicitly this parameter we preserve the original behaviour, otherwise we use KeyValueEncoding.

I am not adding the part about encoding the "key" with the schema because it is not possible in Pulsar to set the schema for the key without KeyValueSchema.
I leave that as a future enhancement